### PR TITLE
Add django-lockdown support

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: admin_client.py [hunk 1]
```diff
@@ -121,13 +122,33 @@ def encode_data(self, data):
     def post_form(self, url, data=None, select=None):
         resp = self.session.get(url)
         soup = self.get_soup(resp)
+        lockdown = soup.select("#lockdown")
+        if lockdown:
+            print('Detected lockdown')
```
[the remark this answers is not in the document]
Comment (jayvdb) [reply admin_client.py:127]: This codebase uses print for console information, warnings, etc.

**Comment by jayvdb** [comment]
@bayersglassey repushed.  Happy to remove the `print` if you feel it isnt useful.

**Comment by jayvdb** [comment]
If you dont consider this useful for the main library, I could achieve the same result by maintaining my own subclass that adds this step.

**Comment by bayersglassey** [comment]
Hey, sorry I didn't notice this PR for... 8 months??
I had intended this repo to be minimalist, so as few dependencies as possible. This PR doesn't add a dependency per se, but it's adding shall we say a conceptual dependency to another library.
I think I'll decline the PR, although if there are changes which could be made to make subclassing easier, let me know.

**Pull request closed without merging**